package.json

```diff
@@ -33,7 +33,7 @@
     "ace-builds": "^1.4.12",
     "actioncable": "^5.2.4-3",
     "chart.js": "^3.1.0",
-    "codemirror-lang-elixir": "^1.0.2",
+    "codemirror-lang-elixir": "https://github.com/sachinraja/codemirror-lang-elixir",
```

Comment:
The package on npm isn't updated to the latest version. I pointed the package to github instead.